packages/block-library/src/heading/block.json

```diff
@@ -29,7 +29,7 @@
 	"supports": {
 		"align": [ "wide", "full" ],
 		"anchor": true,
-		"className": false,
+		"className": true,
```

Comment:
It looks like this change might also mean that the className is being serialized to post content. Was that intentional?

Reply:
See https://github.com/WordPress/gutenberg/pull/42122#issuecomment-1331479800